packages/tools/pluggable-widgets-tools/src/typings-generator/__tests__/outputs/association.ts

```diff
@@ -19,7 +19,7 @@ export interface MyWidgetContainerProps {
 }
 
 export interface MyWidgetPreviewProps {
-    className: string;
+    class: string;
```

Comment:
We should keep both fields. Ideally we mark `className` as deprecated with JSDoc to make users know that this is not recommended to use and `class` have to be used instead.